### ISSUE TITLE
Lidarr not adding album art to finished download

### ISSUE THREAD

**Issue by diabl0w** [lidarr/Lidarr#1746]
**Describe the bug**
Lidarr does not add album art to the the album folder (it **does** add artist art, fanart, etc to the artist folder)

**To Reproduce**
1. Enable kodi metadata
2. enable "tag audio files with metadata"
3. mark as wanted/download/import album

**Logs**
After adding the artist to be monitored, lidarr correctly gets the cover for the appropriate ablum:
```
20-11-14 10:58:59.6|Info|MediaCoverService|Downloading Cover for [9a29e57d-dd1c-35c1-bb05-d220bcb656d6][Lifestylez ov da Poor & Dangerous] https://coverartarchive.org/release/b6f8616c-9d1c-44d1-b8f4-aaf9a3c17f5f/4394279316.jpg
20-11-14 10:58:59.6|Debug|HttpClient|Downloading [https://coverartarchive.org/release/b6f8616c-9d1c-44d1-b8f4-aaf9a3c17f5f/4394279316.jpg] to [/config/MediaCover/Albums/138/cover.jpg]
```
Marked the album as wanted and successfully downloaded via nzbget download client, and lidarr successfuly imported the album EXCEPT it did not add any album art. Obviously skipping a lot of logs here, but I looked at every single line of the trace log and there was nothing relevant for adding the album artwork as a jpeg to the album folder. However the album art was embedded into the tracks, but this is not what is desired (I enabled the kodi metadata option so it should be adding album artwork as a jpeg to the album folder):
```
20-11-14 11:01:43.9|Debug|DiskProvider|Setting permissions: 0644 on /media/music/Big L/Lifestylez ov da Poor & Dangerous (1995)/01 - Put It On.mp3
20-11-14 11:01:44.0|Trace|AudioTagService|Embedding: /config/MediaCover/Albums/138/cover.jpg
```
I think this is probably irrelevant, but I was also seeing these errors in the log for every track that lidarr was moving from the download folder to the library folder as it was processing, just as possible additional info:
```
20-11-14 11:01:44.7|Trace|ConfigService|Using default config value for 'scrubaudiotags' defaultValue:'False'
20-11-14 11:01:44.7|Debug|AudioTagService|Writing tags for [0] /media/music/Big L/Lifestylez ov da Poor & Dangerous (1995)/09 - Lifestylez ov da Poor & Dangerous.mp3
20-11-14 11:01:44.7|Debug|AudioTag|Starting tag write for /media/music/Big L/Lifestylez ov da Poor & Dangerous (1995)/09 - Lifestylez ov da Poor & Dangerous.mp3
20-11-14 11:01:44.7|Trace|ConfigService|Using default config value for 'scrubaudiotags' defaultValue:'False'
20-11-14 11:01:44.7|Trace|EventAggregator|Publishing TrackFileRetaggedEvent
20-11-14 11:01:44.7|Trace|EventAggregator|TrackFileRetaggedEvent -> NotificationService
20-11-14 11:01:44.7|Trace|EventAggregator|TrackFileRetaggedEvent <- NotificationService
20-11-14 11:01:44.7|Trace|EventAggregator|TrackFileRetaggedEvent -> HistoryService
20-11-14 11:01:44.7|Trace|Owin|SQLite error (5): database is locked
```
**System info (please complete the following information):**
 - latest linuxserver docker image

**Comment by diabl0w** [comment]
update, step 2 not even needed... i definitely have another drive with old lidarr downloads where lidarr properly downloaded the album art (as cover.jpg) and it is just not working at all now. the only relevant setting i know is the kodi metadata setting and it is enabled

**Comment by diabl0w** [comment]
here are 2 recent commits i found that seem related to album art processing:
https://github.com/lidarr/Lidarr/commit/9714186243835b274ec78a4bf51053cf79e3a863
https://github.com/lidarr/Lidarr/commit/a085dd38a6d84f04ac24ec543ef37324859f6858

no idea if they are relevant, but just trying to do what i can to help

**Comment by diabl0w** [comment]
Issue looks fixed in v0.8x.. I guess linuxserver/lidarr:latest actually is not latest, but instead was 0.7x
